dbt/model.py

```diff
@@ -133,7 +135,10 @@ def get_project_config(self, project):
         for k in SourceConfig.ExtendDictFields:
             config[k] = {}
 
-        model_configs = project.get('models')
+        if self.node_type == NodeType.Seed:
+            model_configs = project.get('data')
```

Comment:
`data` is just the default seed data location right? should this be `seeds` instead (since the other one is `models`)?